v2/internal/frontend/desktop/darwin/frontend.go

```diff
@@ -86,7 +87,8 @@ func NewFrontend(ctx context.Context, appoptions *options.App, myLogger *logger.
 		} else {
 			appBindings.DB().UpdateObfuscatedCallMap()
 		}
-		assets, err := assetserver.NewAssetServerMainPage(ctx, bindings, appoptions)
+
+		assets, err := assetserver.NewAssetServerMainPage(bindings, appoptions, ctx.Value("assetdir") != nil, myLogger, runtime.RuntimeAssetsBundle)
```

Comment:
Linux and Windows also contain this call in their frontend.go. Those changes are currently missing in the PR.

Reply:
Ah the perils of build tags and IDEs 😭